### PR TITLE
Add 404 page translated

### PR DESCRIPTION
**Pull request by jakeherp**

### CODE REVIEW

File: docs/docs/add-404-page.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Adding a 404 Page
+title: Hinzufügen einer 404 Seite
 ---
 
-To create a 404 page create a page whose path matches the regex `^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`). Most often you'll want to create a React component page at `src/pages/404.js`.
+Um eine 404 Seite zu erstellen, füge eine Seite, die dem Regex `^\/?404\/?$` (`/404/`, `/404`, `404/` oder `404`) entspricht. Üblich ist es dafür eine React Komponente unter `src/pages/404.js` zu erstellen.
```
Comment (gustavpursche) [inline docs/docs/add-404-page.md]:
```suggestion
Um eine 404 Seite zu erstellen, füge eine Seite, die dem Regex `^\/?404\/?$` (`/404/`, `/404`, `404/` oder `404`) entspricht. Üblicherweise erstellst du dafür eine React Komponente in `src/pages/404.js`.
```

File: docs/docs/add-404-page.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-Gatsby ensures that your 404 page is built as `404.html` as many static hosting platforms default to using this as your 404 error page. If you're hosting your site another way you'll need to set up a custom rule to serve this file for 404 errors.
+Gatsby stellt sicher, dass deine 404 Seite als `404.html` erstellt wird, da die meisten statischen Hosting Plattformen diese als Standard verwenden. Wenn du deine Seite anderweitig hostest musst du eine individuelle Regel erstellen, um die Seite für 404 Fehler aufrufbar zu machen.
```
Comment (gustavpursche) [inline docs/docs/add-404-page.md]:
```suggestion
Gatsby stellt sicher, dass deine 404 Seite als `404.html` erstellt wird, da die meisten statischen Hosting Plattformen diese als Standard verwenden. Wenn du deine Seite anderweitig hostest, musst du eine individuelle Regel erstellen, um die Seite für 404 Fehler aufrufbar zu machen.
```

File: docs/docs/add-404-page.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
-Because Gatsby creates this page for you by default, there is no need to configure it in your `gatsby-node.js` file.
+Da Gatsby diese Seite standardmäßig für dich erstellt ist es nicht notwendig diese in `gatsby-node.js` zu konfigurieren.
```
Comment (gustavpursche) [inline docs/docs/add-404-page.md]:
```suggestion
Da Gatsby diese Seite standardmäßig für dich erstellt, ist es nicht notwendig diese in `gatsby-node.js` zu konfigurieren.
```

File: docs/docs/add-404-page.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 
-When developing using `gatsby develop`, Gatsby uses a default 404 page that overrides your custom 404 page. However, you can still preview your 404 page by clicking "Preview custom 404 page" to verify that it's working as expected. This is useful when you're developing so that you can see all the available pages.
+Beim Entwickeln mit `gatsby develop` nutzt Gatsby eine eigene 404 Seite die deine überschreibt. Allerdings kannst du deine 404 Seite ansehen, indem du auf "Preview custom 404 page" klickst um sicherzustellen, dass alles wie gewünscht funktioniert.
```
Comment (gustavpursche) [inline docs/docs/add-404-page.md]:
```suggestion
Beim Entwickeln mit `gatsby develop` nutzt Gatsby eine eigene 404 Seite, welche deine überschreibt. Du kannst du deine 404 Seite allerdings ansehen, indem du auf "Preview custom 404 page" klickst um sicherzustellen, dass alles wie gewünscht funktioniert.
```

**Comment by jakeherp** [comment]
Thanks for your CR @gustavpursche! Addressed all your suggestions.

**Pull request merged**